service/Microsoft.Azure.Devices/test/Microsoft.Azure.Devices.Api.Test/RegistryManagerTests.cs

```diff
@@ -71,14 +72,18 @@ public void ConstructorWithValidConnectionStringTest()
         [TestMethod]
         [TestCategory("CIT")]
         [TestCategory("API")]
-        [Ignore] //Issue #318
```

Comment:
Is this fixing #318 or are there other issues before we can close that?

Reply:
There are more test that were disabled as tracked by #318.  I am only fixing one of the tests (non-WS).